From db9e38650c0b04301dc1e3eeadd9d246d8a3aa2d Mon Sep 17 00:00:00 2001
From: Chris Packham <judge.packham@gmail.com>
Date: Mon, 2 Jun 2025 16:57:25 +1200
Subject: [PATCH] libgloss: use expected prototype for __inln()

Resolve the following build error

  newlib/libgloss/rs6000/mvme-read.c:31:24: error: too many arguments to function '__inln'; expected 0, have 1
    31 |       char *read_end = __inln (buf);
       |                        ^~~~~~  ~~~

Signed-off-by: Chris Packham <judge.packham@gmail.com>
---
 libgloss/rs6000/mvme-read.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/libgloss/rs6000/mvme-read.c b/libgloss/rs6000/mvme-read.c
index 1c823b433..f8c539e76 100644
--- a/libgloss/rs6000/mvme-read.c
+++ b/libgloss/rs6000/mvme-read.c
@@ -14,7 +14,7 @@
  */
 
 extern int inbyte ();
-extern char * __inln ();
+extern char * __inln (char *buf);
 
 /*
  * read  -- read bytes from the serial port. Ignore fd, since
-- 
2.49.0

